src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php

```diff
@@ -13,8 +13,9 @@
 
 use Symfony\Component\HttpKernel\CacheWarmer\WarmableInterface;
 use Symfony\Component\Translation\Translator as BaseTranslator;
-use Symfony\Component\Translation\MessageSelector;
 use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Translation\MessageSelector;
```

Comment:
IIUC This cs fix should not be targeting master to ease the merge of older branches